keyboards/ergodox_ez/keymaps/mac_win_german_programmer/keymap.c

```diff
@@ -0,0 +1,357 @@
+#include "ergodox_ez.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```